tests/unit/data_encoding/test_encoding_decoding.py

```diff
@@ -0,0 +1,42 @@
+"""
+To Test the module message_encoding_decoding.py
+"""
+
+from modules import data_encoding
+
+from modules import position_global
+
+
+def test_encoding_decoding() -> None:
+    """
+    Function to test encoding
+    """
+
+    # Step 1: Create a worker_name and PositionGlobal object
+    worker_name = "communications_worker"  # =3 in Worker_Enum
+    success, original_position = position_global.PositionGlobal.create(
+        latitude=34.24902422, longitude=84.6233434, altitude=27.4343424
+    )
+    if not success:
+        return
+
+    # Step 2: Encode the PositionGlobal object
+    encoded_bytes = data_encoding.message_encoding_decoding.encode_position_global(
+        worker_name, original_position
+    )
+    assert encoded_bytes[0] is True
```

Comment:
Instead of doing this, just do `result, encoded_bytes = ...`, and then `assert result` rather than encoded_bytes[0]. Can you change the other ones also?